### ISSUE TITLE
[question] How to display more details about the requests in the generated report

### ISSUE THREAD

**Issue by kedziorm** [serenity-bdd/serenity-core#2632]
When I prepare the request:

```
SerenityRest.given().config(restAssuredConfig)
```

I provide a certificate in restAssuredConfig. 
How can I display information about the certicate (like CN name) in the generated report?
It seems that it provides all important information (headers, body, cookies), but certificate part is missing.

**Comment by wakaleo** [comment]
```
Serenity.recordReportData().withTitle("Certificate details").andContents("...");
```

**Comment by kedziorm** [comment]
Thank you for

```
Serenity.recordReportData().withTitle("Certificate details").andContents("...");
````

I know this method and I use it quite often.
However, this way it will be reported outside of the request part. Is there any other way?

**Comment by wakaleo** [comment]
No, but have a look at the code and see if you can propose a PR if you think there is a way to include this in the standard reporting.
